acceptor/private.local

```diff
@@ -0,0 +1,8 @@
+"""
```

Comment:
`.local` is needed as an example for `prive.py` file for local development.
So that `private.py` file is in `.gitignore`.